src/backend/PortabilityService.Gateway/PortabilityService.Gateway.csproj

```diff
@@ -0,0 +1,27 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
+
+    <!-- Ocelot is not signed and there's no reason for a service assembly
+        (as opposed to a library) to need a strong name signature -->
+    <SignAssembly>false</SignAssembly>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.ApplicationInsights.AspNetCore" Version="2.2.1" />
+    <PackageReference Include="Microsoft.AspNetCore.All" Version="2.0.5" />
+    <PackageReference Include="ocelot" Version="3.1.9" />
+    <PackageReference Include="Polly" Version="5.3.1" />
+    <PackageReference Include="Serilog.AspNetCore" Version="2.1.0" />
```

Comment:
Is Polly used directly? If it's pulled in by ocelot we could remove it.

Reply:
It is because I'm making my own policies to use. By default, we can use Ocelot-produced Polly policies (and then wouldn't need the Polly dependency), but I'm replacing the default policies with our own.